src/main/java/com/arpnetworking/metrics/mad/model/DefaultRecord.java

```diff
@@ -181,16 +193,28 @@ public Builder setAnnotations(final ImmutableMap<String, String> value) {
             return this;
         }
 
+        /**
+         * The dimension mappings <code>ImmutableMap</code>. Optional. Default is an empty
+         * <code>ImmutableMap</code>. Cannot be null.
+         *
+         * @param value The dimension mappings <code>ImmutableMap</code>
+         * @return This instance of <code>Builder</code>.
+         */
+        public Builder setDimensions(final ImmutableMap<String, String> value) {
+            _dimensions = value;
+            return this;
+        }
+
         // Called by OVal reflectively
         @SuppressFBWarnings("UPM_UNCALLED_PRIVATE_METHOD")
-        private boolean validateAnnotations(final ImmutableMap<String, String> annotations) {
-            if (Strings.isNullOrEmpty(annotations.get(Key.HOST_DIMENSION_KEY))) {
+        private boolean validateDimensions(final ImmutableMap<String, String> dimensions) {
+            if (Strings.isNullOrEmpty(dimensions.get(Key.HOST_DIMENSION_KEY))) {
```

Comment:
Same question raised in schema PR.  Will make sure this code is updated to follow whatever is decided there